aspnetcore/migration/50-to-60.md

```diff
@@ -122,6 +122,37 @@ There are a few changes to the other files generated for the Web App template:
 
 For more details on the new hosting model, see the [Frequently asked questions](#faq) section. For more information on the adoption of NRTs and .NET compiler null-state analysis, see the [Nullable reference types (NRTs) and .NET compiler null-state static analysis](#nullable-reference-types-nrts-and-net-compiler-null-state-static-analysis) section.
 
+### Apps migrating to 6.0 don't need to use the new minimal hosting model
+
+Using `Startup` and the [Generic Host](xref:fundamentals/host/generic-host) used by the ASP.NET Core 3.1 and 5.0 templates is fully supported.
```

Comment:
Should this just be a note/warning at the top of `## New hosting model`? I think it might be less likely to be missed that way.